### PR TITLE
fix picture missing (Issue #1305)

### PR DESCRIPTION
**Pull request by Dashlorde**
Change picture name for picture to be displayed on the page, see [router config page](http://open-learning-exchange.github.io/#!pages/techgenius/tg-router-config.md)

### CODE REVIEW

**Comment by ajGingrich** [comment]
looks like a good catch. Do you have a link to the rawgit so we can review the updated picture easily?

**Comment by Dashlorde** [comment]
see [rawgit link](https://rawgit.com/Dashlorde/Dashlorde.github.io/configuring-router/#!pages/techgenius/tg-router-config.md)

File: pages/techgenius/tg-router-config.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ _Via Command prompt_. Open the CMD window (press Windows button + R or search th
  
 _Via the Graphical User Interface_. Open Control Panel > Network and Sharing Center > Wi-Fi Connections (click link) > Details to get this dialogue box:
 
-![Windows GUI IP screen shot](images/tg-wifi-gui-config.png)
+![Windows GUI IP screen shot](images/wifi-gui-config.png)
```
Comment (dogi) [inline pages/techgenius/tg-router-config.md]: @Dashlorde think we want to change the file name by moving the file to the new name
`git mv wifi-gui-config.png tg-wifi-gui-config.png`

**Comment by Dashlorde** [comment]
I have changed the picture filename.

**Pull request merged**